pandas/tests/io/test_feather.py

```diff
@@ -102,8 +104,8 @@ def test_read_columns(self):
 
     def test_unsupported_other(self):
 
-        # period
-        df = pd.DataFrame({"a": pd.period_range("2013", freq="M", periods=3)})
```

Comment:
Since feather now exactly maps to the Arrow memory, periods are now supported (since Period is supported in the pandas->pyarrow.Table conversion)

Reply:
that answers my question above, never mind

Reply:
OK!